frontend/src/Browse.js

```diff
@@ -98,7 +99,7 @@ function Browse() {
           }
         </tbody>
       </table>
-    </div>
+    </div> </>
```

Comment:
What are the empty brackets for

Reply:
It did allow me to just add <Nav /> on its own. Either I put it everything within empty brackets. I chose empty brackets Nav and other page components dont ever jumbled due to being in the same div